From a5f59bfeed1d486a81bebc85171210909d2c981f Mon Sep 17 00:00:00 2001
From: Ritesh Raj Sarraf <rrs@debian.org>
Date: Sat, 16 Mar 2013 16:10:36 +0530
Subject: [PATCH 15/20] Don't hardcode the path

There's no need to hardcode the path. From my tests, it shows that udev
does a lookup in /lib/udev/ anyways.

This will also help in cases where udev/systemd stores rules in
/usr/lib/
---
 etc/rules/99-laptop-mode.rules | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/etc/rules/99-laptop-mode.rules b/etc/rules/99-laptop-mode.rules
index 8ff8218..0565644 100644
--- a/etc/rules/99-laptop-mode.rules
+++ b/etc/rules/99-laptop-mode.rules
@@ -1,3 +1,3 @@
-ACTION=="change", SUBSYSTEM=="power_supply", RUN+="@LIB_D@/udev/lmt-udev auto"
-ACTION=="add|remove", SUBSYSTEM=="machinecheck", RUN+="@LIB_D@/udev/lmt-udev auto"
-ACTION=="add|remove", SUBSYSTEM=="usb", RUN+="@LIB_D@/udev/lmt-udev force modules=usb-autosuspend devices=%k"
+ACTION=="change", SUBSYSTEM=="power_supply", RUN+="lmt-udev auto"
+ACTION=="add|remove", SUBSYSTEM=="machinecheck", RUN+="lmt-udev auto"
+ACTION=="add|remove", SUBSYSTEM=="usb", RUN+="lmt-udev force modules=usb-autosuspend devices=%k"
-- 
1.8.1.5

